testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -6566,9 +6566,13 @@ public void testCreateFunction()
         assertUpdate("CREATE FUNCTION " + recursive1 + "(x integer) RETURNS integer RETURN x");
         assertUpdate("CREATE FUNCTION " + recursive2 + "(x integer) RETURNS integer RETURN " + recursive1 + "(x)");
         assertUpdate("CREATE OR REPLACE FUNCTION " + recursive1 + "(x integer) RETURNS integer RETURN " + recursive2 + "(x)");
-        assertQueryFails("SELECT " + recursive1 + "(42)", "line 3:8: Recursive language functions are not supported: " + recursive1 + "\\(integer\\):integer");
+        assertQueryFails("SELECT " + recursive1 + "(42)", "Recursive language functions are not supported: " + recursive1 + "\\(integer\\):integer");
+        assertQueryFails("SELECT " + recursive2 + "(42)", "Recursive language functions are not supported: " + recursive2 + "\\(integer\\):integer");
         assertUpdate("DROP FUNCTION " + recursive1 + "(integer)");
         assertUpdate("DROP FUNCTION " + recursive2 + "(integer)");
+
+        // verify error location reporting
+        assertQueryFails("CREATE FUNCTION bad() RETURNS varchar RETURN 8", "line 1:46: Value of RETURN must evaluate to varchar \\(actual: integer\\)");
```

Comment:
Doesn't need to be in BCT, provided that it doesn't exercise the connector

Reply:
Good point, this will fail before we actually try to create it